tools/build_executable.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--temp_dir", required=True)
```

Comment:
set this to be "tmp" as a default, or to os.path.join(os.path.dirname(os.path.realpath(__file__), "tmp") to match the existing convention for temp_dir in our code

Reply:
Done